inst/rmarkdown/templates/mortality/skeleton/skeleton.Rmd

```diff
@@ -516,24 +516,46 @@ descriptive(study_data_cleaned,
 Compare the proportions in each age group between your sample and the source population. 
```

Comment:
@zkamvar - added in fishers exact test to compare for each age group if the proportion sampled differed significantly from the actual population. 
Not sure this is the most appropriate approach - I know you can do a regular chisq test on the whole table, but dont feel that gives enough information on where the bias is (can infer from the residuals, but thats messy); also can be thrown off by one cell being unexpected...
Also remember we had discussions about correcting for multiple testing... 
Any better ideas?
Also code needs massively simplifying! (i think? - or can we leave as is with more explanation?)

Reply:
I believe we can use a binomial test for this and then correct for multiple comparisons with `p.adjust()`